### PR TITLE
cc11

### PR DESCRIPTION
**Pull request by baxance**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x} Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: java/challenges/lib/src/main/java/challenges/stacksandqueues/PseudoQueue.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
+package challenges.stacksandqueues;
+
+import java.util.NoSuchElementException;
+
+public class PseudoQueue {
+  public Node top;
```
Comment (Mattpet26) [inline java/challenges/lib/src/main/java/challenges/stacksandqueues/PseudoQueue.java:6]: Pseudoqueue doesn't have a top node! Only the two stacks within it.

File: java/challenges/lib/src/main/java/challenges/stacksandqueues/PseudoQueue.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    stackTwo = new Stack();
+  }
+
+  public void pseudoEnqueue(Object val){
+      stackOne.push(val);
+    }
+
+  public Object pseudoDequeue(){
+    Object val;
+    if (stackTwo.top == null){
```
Comment (Mattpet26) [inline java/challenges/lib/src/main/java/challenges/stacksandqueues/PseudoQueue.java:22]: instead of running a double if(statement), you can wrap them both in one by using if(thing1 && thing2)
